### ISSUE TITLE
unable to search or text in pdf.

### ISSUE THREAD

**Issue by NSP-0123456** [tgotic/xPDFSearch#1]
ref: #https://www.ghisler.ch/board/viewtopic.php?p=456273#p456273

I was using version 1.11 all was ok, i just switched to 1.41 but I have an issue with xpdfsearch 1.41, if i want to search for text in pdf. .

on tc9 it works on 32bit and not in 64bit.
on tc 11.03 none of the version works.

I search in background using plugin xpdfsearch.text contain "myString"  the plugin seems to crash as the search windows is closed without notice.
The version 1.11 was working correctly on both 32 an 64bit.

I use TotalCmd 11.03 in plain English and search is usually done in background. 
// I cannot disable antivirus as it is managed by company policies.
I have one line in xpdfsearch.ini in same folder as plugin (%commander_path%\addon\xpdfsearch )
```
[xPDFSearch]
NoCache=1
```
I do not have any xpdfrc file.
Do you have any debug version with log or ability to see detailed informations ?

Currently i see in nirsof simpleProgramDebugger : 
xpdfsearch create a thread and it exit just after. 
Critical error detected c0000374
Exception Code: 0xc0000374, Parameters: , 0xd83b38b0

**Comment by tgotic** [comment]
Bug in GString::del() that might cause crash on text search is fixed in v1.42.

**Comment by NSP-0123456** [comment]
[06_Managing, building, and deploying SSC code.zip](https://github.com/user-attachments/files/15775758/06_Managing.building.and.deploying.SSC.code.zip)
This file seems to break the search. It is ok to search with SumatrapDF.

**Comment by tgotic** [comment]
Fixed in v1.42
